### ISSUE TITLE
Proposal: Fetch BTS network data through web server?

### ISSUE THREAD

**Issue by grctest** [BTS-CM/bitshares-solid-ui#1]
Solid-start enables us to run server side code and return the data to the client: https://start.solidjs.com/api/createServerData

We could plausibly use this to fetch data from the network.

e.g. on the explore page showing the block info, we could be fetching this data on the server and returning it to the user.

When broadcasting to the network however this would be performed client-side, to reduce MITM risks.

Advantages:
* Reduction in duplicate requests to full node servers
* Less client side code to run (faster, less downloads, more private).
* Cache-able requests

Disadvantages:
* A potential perception of less direct connection to the DEX
* Full nodes may need to approve greater connection limits for the web wallet making many calls
* Potentially a larger cloud computing bill as computations performed on server, rather than client-side.

Thoughts?

**Comment by sschiessl-bcp** [comment]
You can already do MITM by providing the client with wrong data. It's a strategic choice like you said.

**Comment by grctest** [comment]
True, I figure it could be worth it to reduce the duplicate network calls and client side external downloads
